pkgs/tools/misc/taoup/default.nix

```diff
@@ -0,0 +1,63 @@
+{ lib, stdenv, fetchFromGitHub, ruby, bundlerEnv, bash, ncurses }:
+let
+  gems = bundlerEnv {
+    name = "taoup-env";
+    inherit ruby;
+    gemdir = ./.;
+  };
+
+in
+stdenv.mkDerivation rec {
+  pname = "taoup";
+  version = "1.1.13";
+
+  src = fetchFromGitHub {
+    owner = "globalcitizen";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1qwx88pzkyznzg7z55qksxrw3vh86zz9amr9w78i227ahrm364bx";
+  };
+
+  buildInputs = [ gems ruby ncurses ];
+
+  patches = [
+    # Pre-create a cache within this derivation
+    ./cachedir.patch
+    # Remove the need to test for `tput`, depend on ncurses directly
+    ./tput.patch
+  ];
+
+  postPatch = ''
+    substituteInPlace taoup --subst-var-by ncurses ${ncurses}
+    substituteInPlace taoup-fortune \
+      --subst-var-by out $out \
+      --replace "/bin/bash" "${bash}/bin/bash"
+  '';
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  installPhase = ''
+        mkdir -p $out/{bin,lib/taoup}
```

Comment:
This section looks like it has been indented too far, is that intentional?

Reply:
Yeah, it was indented by `nixpkgs-fmt` though - let me see if this can be de-indented :muscle:

Reply:
Fixed this by keeping the heredoc on the same indent level as others - doesn't seem to affect the result.  Thanks!